app/furniture/marketplace/carts/_footer.html.erb

```diff
@@ -8,7 +8,7 @@
   <tr>
     <td class="text-left px-3 py-3.5 text-right" colspan="8">
       <%= render "marketplace/carts/total", cart: cart %>
-      <%- if cart.ready_for_delivery? %>
+      <%- if cart.ready_for_checkout? %>
         <%= button_to("Checkout", cart.location(child: :checkout), data: { turbo: false }) %>
```

Comment:
I know this is not part of your PR, but it's been bugging me for a bit: this should be "Check Out", not "Checkout" (i.e. the imperative of a verb, not a noun).

Reply:
I get that, I'm down for shifting towards `Check Out` !